Userland/Libraries/LibWeb/Animations/KeyframeEffect.cpp

```diff
@@ -871,6 +871,27 @@ void KeyframeEffect::visit_edges(Cell::Visitor& visitor)
         visitor.visit(keyframe);
 }
 
+static CSS::RequiredInvalidationAfterStyleChange compute_required_invalidation(HashMap<CSS::PropertyID, NonnullRefPtr<CSS::StyleValue const>> const& old_properties, HashMap<CSS::PropertyID, NonnullRefPtr<CSS::StyleValue const>> const& new_properties)
+{
+    CSS::RequiredInvalidationAfterStyleChange invalidation;
+    auto old_and_new_properties = MUST(Bitmap::create(to_underlying(CSS::last_property_id) + 1, 0));
+    for (auto property_id : old_properties.keys())
+        old_and_new_properties.set(to_underlying(property_id), 1);
+    for (auto property_id : new_properties.keys())
+        old_and_new_properties.set(to_underlying(property_id), 1);
+    for (auto i = to_underlying(CSS::first_property_id); i <= to_underlying(CSS::last_property_id); ++i) {
+        if (!old_and_new_properties.get(i))
+            continue;
```

Comment:
This would be even less work if we iterated over the set bits in `old_and_new_properties`. But I don't think we have a way to do that, so we can let it slide for now :sweat_smile: